app/services/flow/flows/citizen_dependants.rb

```diff
@@ -17,11 +17,16 @@ class CitizenDependants < FlowSteps
           path: ->(_, dependant) { urls.citizens_dependant_relationship_path(dependant) },
           forward: ->(_, dependant) { dependant.adult_relative? || dependant.eighteen_or_less? ? :dependants_income : :dependants_education }
         },
+        dependants_assets_values: {
+          # path: ->(_, dependant) { urls.citizens_dependant_assets_values_path(dependant) },
+          forward: :has_other_dependants
```

Comment:
I think it makes sense to keep this one (perhaps add a TODO).

The final flow will not work without each step having a `:path` defined. I expect it's only being commented out because the code coverage sees this as unused.

I think that it makes sense that the dev who builds the step in the flow works out the correct path, rather than the next dev whose only involvement is to use it in their `:forward` declaration from another step.

Reply:
I agree with Rob

Reply:
yep, makes sense.